src/collections/_documentation/development/sdk-dev/event-payloads/threads.md

```diff
@@ -27,12 +27,14 @@ exception and Sentry will connect the two.
 
 `crashed`:
 
-: _Optional_. A flag indicating whether the thread crashed. Defaults to `false`.
+: _Optional_. A flag indicating whether this thread has caused the error. It does not necessarily indicate the application terminated. Defaults to `false`.
+It may be seen as `crashed: yes` in the Issues page.
```

Comment:
```suggestion
It may display as `crashed: yes` on the Issues page.
```

Reply:
Both options here ("may be seen" and "may display") suggest that it may or may not display like that. Is there a case where the event has `crashed: true` and doesn't display `crashed: yes` in Sentry?!

Why do we say "may"?

Reply:
If the usage of "may" was future-proofing in case of UI changes, then maybe we should not mention it altogether?

Reply:
Great question! I was assuming that we included the word "may" because it was uncertain? Happy to take your lead on this and omit the word "may." @marandaneto do you know if there's a reason to emphasize "may?"

Reply:
@MimiDumpling tbh no, it's just a synonym of `can` that didn't work well, so the meaning would be: `can be seen/it can be viewed`, I've changed to `it displays as...` :)

Reply:
@bruno-garcia true, we can talk about this on TSC again, I generally agree, but still think the current status of docs would need to be changed, either by this PR or other, this is a very recurrent problem.
I'll keep this PR open then.